crates/uv-distribution-filename/src/wheel.rs

```diff
@@ -267,6 +314,112 @@ impl Serialize for WheelFilename {
     }
 }
 
+/// A [`SmallVec`] type for storing tags.
+///
+/// Wheels tend to include a single language, ABI, and platform tag, so we use a [`SmallVec`] with a
+/// capacity of 1 to optimize for this common case.
+pub type TagSet<T> = smallvec::SmallVec<[T; 3]>;
```

Comment:
Not sure if it's still worth doing this. Debatable. (Only used in the large variant now, which is boxed anyway.)

Reply:
I'm pro-smallvec, for small types it has always been faster in the cases i tried, and the codepath is allocation-heavy.